### PR TITLE
feat: relations between entities

### PR DESCRIPTION
**Pull request by leomoraiscam**
In this PR, the relationship between entities and the recovery of this data through associations was implemented and some permissions were added to more than users

### CODE REVIEW

File: src/repositories/vottes.repository.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const { Votes } = require('../models');
+const { Votes, User } = require('../models');
```
Comment (rafaelrodriguesioasys) [inline src/repositories/vottes.repository.js]: você não está usando esse model.

File: src/repositories/vottes.repository.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { Votes } = require('../models');
```
Comment (rafaelrodriguesioasys) [inline src/repositories/vottes.repository.js]: alterar o nome do arquivo

Comment (leomoraiscam) [reply src/repositories/vottes.repository.js]: corrigido.

File: src/repositories/vottes.repository.js (hunk 1 above)
Comment (leomoraiscam) [reply src/repositories/vottes.repository.js]: corrigido.

**Review by rafaelrodriguesioasys — APPROVED** [review]

**Pull request merged**